common/src/main/java/org/opensearch/ml/common/ModelAccessIdentifier.java

```diff
@@ -0,0 +1,42 @@
+/*
+ *
+ *  * Copyright OpenSearch Contributors
+ *  * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package org.opensearch.ml.common;
+
+import lombok.Getter;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum ModelAccessIdentifier {
```

Comment:
Minor: How about change the name to "ModelAccessMode"?

Reply:
I see the sample test request has both `model_access_identifier` and `backend_roles`
```
{
    "name": "model_group_restricted",
    "description": "This is an example description",
    "model_access_identifier": "restricted",
    "backend_roles": [
        "role-2"
    ],
    "tags": {
        "key1": "value1",
        "key2": "value2"
    }
}
```
If backend role is not empty, that means the model is "restricted", right? I think we can simplify the input by removing the `model_access_identifier` for this use case.